### ISSUE TITLE
Testing Environment does not enforce sandbox environment usage

### ISSUE THREAD

**Issue by addisonlynch** [addisonlynch/iexfinance#137]
## Summary (include Python version)

The test suite does not currently enforce the use of the sandbox environment. As such, it is possible to run the test suite using production data, which could cause an extremely high message count to be used.

**Comment by jackmoody11** [comment]
I think something like the following might work:

```python
@pytest.fixture(scope='session')
def mock_env_iex_api_version(monkeypatch):
    monkeypatch.setenv("IEX_API_VERSION", "iexcloud-sandbox")

@pytest.fixture(scope='session')
def mock_env_iex_token(monkeypatch):
    monkeypatch.setenv("IEX_TOKEN", "<test token here>")
```

**Comment by nckturner** [comment]
I think this should be the highest priority issue, I just inadvertently ran the tests a few times with my production API key and it cost me ~16k messages.

**Comment by Olshansk** [comment]
The changes in iexfinance/base.py in https://github.com/addisonlynch/iexfinance/pull/237 should handle this..

**Comment by addisonlynch** [comment]
Closed via #237.
